helper-cli/src/main/kotlin/commands/packageconfig/CreateCommand.kt

```diff
@@ -140,10 +159,37 @@ internal class CreateCommand : CliktCommand(
                 emptyList()
             }
         )
-}
 
-private fun ScanResult.getFindingPaths(): Set<String> =
-    mutableSetOf<String>().apply {
-        summary.licenseFindings.mapTo(this) { it.location.path }
-        summary.copyrightFindings.mapTo(this) { it.location.path }
+    private fun ScanResult.getFindingPaths(): Set<String> {
+        val nonOffendingLicenses = getNonOffendingLicenses()
+
+        return mutableSetOf<String>().apply {
+            summary.licenseFindings.filter {
+                nonOffendingLicenses.isEmpty() || !nonOffendingLicenses.containsAll(it.license.decompose())
+            }.mapTo(this) { it.location.path }
+            summary.copyrightFindings.filter {
+                nonOffendingLicenses.isEmpty()
+            }.mapTo(this) { it.location.path }
+        }
     }
+
+    private fun getNonOffendingLicenses(): Set<SpdxSingleLicenseExpression> {
+        // Filter blanks to allow passing an empty list as argument to simplify caller code.
+        if (nonOffendingLicenseCategories.filterNot { it.isBlank() }.isEmpty()) return emptySet()
```

Comment:
Where are the blanks coming from? Could this already be filtered by using `convert` on the option definition?

Reply:
The blanks are coming from [here](https://github.com/oss-review-toolkit/orthw/pull/42/commits/50f4068612d0df678cd525a30e707528f4b0fcf6#diff-5c2c575574618fe09803be3a596038a591a7f4526285a1f7be0faefe7061f42eR241).

I tried quite a bit but haven't found a way to filter them out by using `convert`. IIUC convert is supposed to change the `type`. If you have an idea how could be done...?!

Reply:
No, I was just wondering. But I realized now the statement can be simplified: `nonOffendingLicenseCategories.all { it.isBlank() }`